src/pg_query_deparse.c

```diff
@@ -323,6 +324,9 @@ static void deparseExpr(StringInfo str, Node *node)
 		case T_GroupingFunc:
 			deparseGroupingFunc(str, castNode(GroupingFunc, node));
 			break;
+		case T_List:
+			deparseTList(str, castNode(List, node));
+			break;
```

Comment:
Instead of doing this on the list type (which is used in a lot of places, not always in a context where `;` is the correct separator), can we do this on the parent node? (i.e. check if `CreateFunctionStmt` `sql_body` is a list, and then iterate over it there)

Reply:
Rebased and checked in CreateFunctionStmt
PR: https://github.com/pganalyze/libpg_query/pull/197